src/CMakeLists.txt

```diff
@@ -71,7 +71,7 @@ generate_export_header(InfluxDB)
 
 target_include_directories(InfluxDB
   PUBLIC
-    $<INSTALL_INTERFACE:include>
+    $<INSTALL_INTERFACE:include/${HEADER_INSTALL_SUBDIR}>
```

Comment:
By setting `CMAKE_INSTALL_INCLUDEDIR`, user might have redirected the installation dir already - it is just not covered by the hard-coded `include` here.

Reply:
What would need to be changed for that?

Reply:
ping @dg0yt